assets/dev/js/editor/components/template-library/views/parts/templates.js

```diff
@@ -59,7 +59,7 @@ TemplateLibraryCollectionView = Marionette.CompositeView.extend( {
 	},
 
 	getChildView( childModel ) {
-		if ( 'remote' === childModel.get( 'source' ) ) {
+		if ( childModel.get( 'source' ).startsWith( 'remote' ) ) {
```

Comment:
is there a way we either localize the script to indicate which sources are "remote", or add childModel.get( 'source' ).isRemote or something a little more general?

Reply:
How does this all work together? Why do you need a new "source"? Are you calling a totally new API?

Reply:
we are adding a remote-source that extends the current Source_Remote class and registers etc'...

Reply:
@nicoladj77 maybe use the same filter here as well?
Is `elementor.hooks` available here?
```suggestion
		const isRemote = elementor.hooks.applyFilters( 'templates/modal/active-source', childModel.get( 'source' ) !== 'local', activeSource );
		if ( isRemote ) {
```